src/pages/FlowRun/Details-Tile.vue

```diff
@@ -343,41 +276,29 @@ export default {
           </v-list-item>
           <LabelEdit type="flowRun" :flow-run="flowRun" />
         </v-list>
-      </v-fade-transition>
-
-      <v-fade-transition hide-on-leave>
-        <div
-          v-if="tab === 'parameters'"
-          class="text-body-2 appForeground rounded-sm pa-5 code-block"
-        >
+      </v-tab-item>
+      <v-tab-item :key="tabs.parameters">
+        <div class="text-body-2 appForeground rounded-sm pa-5 code-block">
           {{ formatJson(flowRunParams) }}
         </div>
-      </v-fade-transition>
-
-      <v-fade-transition hide-on-leave>
-        <div
-          v-if="tab === 'context'"
-          class="text-body-2 appForeground rounded-sm pa-5 code-block"
-        >
+      </v-tab-item>
+      <v-tab-item :key="tabs.context">
+        <div class="text-body-2 appForeground rounded-sm pa-5 code-block">
           {{ formatJson(flowRun.context) }}
         </div>
-      </v-fade-transition>
-
-      <v-fade-transition hide-on-leave>
-        <div
-          v-if="tab === 'run_config'"
-          class="text-body-2 appForeground rounded-sm pa-5 code-block"
-        >
+      </v-tab-item>
+      <v-tab-item :key="tabs.run_config">
+        <div class="text-body-2 appForeground rounded-sm pa-5 code-block">
           {{ formatJson(flowRun.run_config) }}
         </div>
-      </v-fade-transition>
-    </v-card-text>
+      </v-tab-item>
+    </v-tabs-items>
   </v-card>
 </template>
 
-<style lang="scss" scoped>
+<style lang="scss">
 .card-content {
-  max-height: 254px;
+  max-height: calc(254px - var(--v-tabs-height));
```

Comment:
I think since your `v-tabs` has a `v-if` on it you need to conditionally subtract `--v-tabs-height`

Reply:
good catch, although upon closer inspection it seems that class isn't actually used in this template